components/kyma-environment-broker/internal/process/deprovisioning/avs_evaluations.go

```diff
@@ -53,8 +54,15 @@ func (ars *AvsEvaluationRemovalStep) Run(operation internal.Operation, logger lo
 	}
 	operation, err = ars.delegator.DeleteAvsEvaluation(operation, logger, ars.externalEvalAssistant)
 	if err != nil {
-		return ars.deProvisioningManager.RetryOperation(operation, "error while deleting avs external evaluation", err, 10*time.Second, 10*time.Minute, logger)
+		logger.Warnf("unable to delete external evaluation: %s", err.Error())
+		return ars.deProvisioningManager.RetryOperationWithoutFail(operation, "error while deleting avs external evaluation", 10*time.Second, 1*time.Minute, logger)
 	}
-	return operation, 0, nil
+
+	newOperation, err := ars.operationsStorage.UpdateOperation(operation)
```

Comment:
Hm, inside the ars.delegator.DeleteAvsEvaluation there is a call to UpdateOperation, once the evaluation could successfully be deleted / processed. This may not be needed here